src/test/java/nextstep/member/acceptance/MemberAcceptanceTest.java

```diff
@@ -72,6 +78,15 @@ void deleteMember() {
     @DisplayName("내 정보를 조회한다.")
     @Test
     void getMyInfo() {
+        // given
+        회원_생성_요청(EMAIL, PASSWORD, AGE);
+
+        String accessToken = 토큰_기반_로그인_요청(EMAIL, PASSWORD).jsonPath().getString("accessToken");
```

Comment:
```suggestion
        var accessToken = 토큰_기반_로그인_요청(EMAIL, PASSWORD).jsonPath().getString("accessToken");
```
전체적으로 var로 통일해주는 것도 좋을 것 같네요 : )